CMakeLists.txt

```diff
@@ -220,6 +223,7 @@ endif()
 
 if(NOT HAIKU AND NOT MSVC AND NOT PSP2)
 	set(M_LIBRARY m)
+	set(PKGCFG_LDFLAGS_PRIV "${PKGCFG_LDFLAGS_PRIV} -lm")
```

Comment:
This should probably be done by looping over `OS_LIB` at the end of the platform support block (right before the feature dependencies block)

Reply:
Are you planning to do anything about this comment?